pkg/kdump/kdump-view.jsx

```diff
@@ -485,7 +486,12 @@ export class KdumpPage extends React.Component {
                         <Title headingLevel="h2" size="3xl">
                             {_("Kernel crash dump")}
                         </Title>
-                        {kdumpSwitch}
+                        <>
```

Comment:
Does the extra `<>` do anything? I think it does not actually get rendered, that's just a container for functions returning multiple elements.